Wrap a test that was failing with exception handling in order to pass testing.
https://github.com/lit26/finvizfinance/commit/ddd263736be278c8834242679c08c804d17ba1c0
--- a/finvizfinance/quote.py	2023-02-03 21:17:45.000000000 -0600
+++ b/finvizfinance/quote.py	2023-05-29 17:56:18.702135822 -0500
@@ -133,12 +133,20 @@
         table = self.soup.find("table", class_="fullview-title")
         rows = table.findAll("tr")
 
-        fundament_info["Company"] = rows[1].text
-        (
-            fundament_info["Sector"],
-            fundament_info["Industry"],
-            fundament_info["Country"],
-        ) = rows[2].text.split(" | ")
+        try:
+            fundament_info["Company"] = rows[1].text
+            (
+                fundament_info["Sector"],
+                fundament_info["Industry"],
+                fundament_info["Country"],
+            ) = rows[2].text.split(" | ")
+        except IndexError:
+            (_, fundament_info["Company"], _) = rows[0].text.split(' | ')
+            (
+                fundament_info["Sector"],
+                fundament_info["Industry"],
+                fundament_info["Country"],
+            ) = rows[1].text.split(" | ")
 
         fundament_table = self.soup.find("table", class_="snapshot-table2")
         rows = fundament_table.findAll("tr")
